### PR TITLE
Update base docker image for EPR

### LINKED ISSUE #119: Docker base image of package-registry is not updated
It looks that the `distribution:snapshot` image, which is the base image for package-registry is not pulled during `elastic-package stack update`.

### PR DESCRIPTION
**Pull request by mtojek**
Closes #119

### CODE REVIEW

**Comment by mtojek** [comment]
I tried to run it locally and it failed with:

```
Error: error running package system tests: could not add data stream config to policy: could not add package to policy; API status code = 500; response body = {"statusCode":500,"error":"Internal Server Error","message":"Stream template not found, no datasets"}
```

**Comment by mtojek** [comment]
It seems that Kibana changes around data stream are present, but I'm confused with shuffled fields in ES documents:

```
[2020-10-05T11:57:25.101Z]              "source": {

[2020-10-05T11:57:25.101Z]                  "geo": {
[2020-10-05T11:57:25.101Z]                      "continent_name": "Europe",
[2020-10-05T11:57:25.101Z]                      "region_iso_code": "DE-RP",
[2020-10-05T11:57:25.101Z]                      "city_name": "Germersheim",
[2020-10-05T11:57:25.101Z] +                    "country_iso_code": "DE",
[2020-10-05T11:57:25.101Z] +                    "country_name": "Germany",
[2020-10-05T11:57:25.101Z]                      "region_name": "Rheinland-Pfalz",
[2020-10-05T11:57:25.101Z]                      "location": {
[2020-10-05T11:57:25.101Z]                          "lon": 8.3639,
[2020-10-05T11:57:25.101Z]                          "lat": 49.2231
[2020-10-05T11:57:25.101Z] -                    },
[2020-10-05T11:57:25.101Z] -                    "country_iso_code": "DE"
[2020-10-05T11:57:25.101Z] +                    }
```

I will restart the job and we'll see...

**Comment by mtojek** [comment]
> I will restart the job and we'll see...

It seems that the order is stable here, but I had to update generated files.

File: internal/stack/todo.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package stack
+
+// TODO compose options
+// TODO stack options
```
Comment (ycombinator) [inline internal/stack/todo.go:8]: Is this file intentional?

Comment (mtojek) [reply internal/stack/todo.go]: I forgot to remove this one, all TODO items are done.

File: internal/docker/docker.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import (
+	"os"
+	"os/exec"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/elastic-package/internal/logger"
+)
+
+// Pull method downloads the latest available revision of the image.
```
Comment (ycombinator) [inline internal/docker/docker.go:16]: Nit: this is a function not a method.

**Review by ycombinator — APPROVED** [review]
LGTM. Nice code cleanup as part of the change!

**Comment by ycombinator** [comment]
>  I'm confused with shuffled fields in ES documents.

Most likely because of this recent change: https://github.com/elastic/elasticsearch/pull/62915
